boolean/boolean.py

```diff
@@ -580,7 +580,7 @@ def get_symbols(self):
         Include recursively subexpressions symbols.
         This includes duplicates.
         """
-        return [s for s in self.get_literals() if isinstance(s, Symbol)]
+        return [s if isinstance(s, Symbol) else s.args[0] for s in self.get_literals()]
```

Comment:
I don't think so, since all literals are either Symbol or Not instances, and Not has one argument.